tests/integration.tests.ps1

```diff
@@ -0,0 +1,7 @@
+Describe "Bicep Module Integration Tests" {
+    Context "Dummy Module" {
+        It "should be true" {
+            $true | should -Be $true
+        }
+    }
+}
```

Comment:
What's the purpose of this?

Reply:
It's just some scaffolding for when we want to run the module tests are part of the build - I'll add a comment.